src/HOCON.Tests/QuotedString.cs

```diff
@@ -26,7 +26,7 @@ public void Bugfix_190_should_unquote_quotedstrings_on_parse()
             ";
 
             var parsed = Parser.Parse(hocon);
-            var unwrapped = parsed.GetObject("adapters").ToDictionary(x => x.Key, v => v.Value.GetString());
+            var unwrapped = parsed["adapters"];
```

Comment:
This is 100000x better than what we had before.

Reply:
`HoconObject` inherits from `ImmutableSortedDictionary`, so there's no need to "unwrap" an object now :)